pygradle-plugin/src/integTest/groovy/com/linkedin/gradle/python/plugin/testutils/ExecUtils.groovy

```diff
@@ -26,6 +26,10 @@ import java.nio.file.Path
 class ExecUtils {
 
     static String run(Path path) {
+        return run(path, new String[0])
+    }
```

Comment:
I'm not entirely sure what this does.  It looks like it adds an empty string to the command line args, but it's not clear to me what the effect, how it's used or how it impacts the fix.